macros/sql/generate_series.sql

```diff
@@ -1,4 +1,8 @@
 {% macro get_powers_of_two(upper_bound) %}
+    {{ adapter.dispatch('get_powers_of_two', packages = dbt_utils._get_utils_namespaces())(upper_bound) }}
```

Comment:
This is causing the error in CI right now:
```suggestion
    {{ return(adapter.dispatch('get_powers_of_two', packages = dbt_utils._get_utils_namespaces())(upper_bound)) }}
```
We need to wrap this in `return()` to preserve the datatype (int) returned by `default__get_powers_of_two`. Otherwise, it's taken as a string.

You can think of the `dispatch` macro as one more gateway through which the macro's call + args need to pass one way, and through which the returned value passes back.

It's not a bad idea to wrap _all_ `adapter.dispatch` calls in `return()`, to be honest.

Reply:
Hi Jeremy,

After wrapping all `adapter.dispatch` with `return()`, we are encountering the error below

![image](https://user-images.githubusercontent.com/59982185/103679177-c1edfc80-4f41-11eb-85e1-d196508166ea.png)

Do you have an idea for how to fix this? Thank you!